### PR TITLE
Profileeditanddelete

### PR DESCRIPTION
**Pull request by ChristophLeuker**
add a delete and edit function for player profiles that works with the MongoDB.

### CODE REVIEW

File: components/profileCard/index.js [hunk 1]
```diff
@@ -9,19 +10,41 @@ const Listitem = styled.li`
...
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  right: 20px;
+  gap: 5px;
+`;
+
 export default function ProfileCard({ name, email }) {
   return (
     <>
```
Comment (rueveydaakuel) [inline components/profileCard/index.js:57]: It is not necessary to use an empty fragment (<>...</>) around the Listitem. You can simply return the Listitem itself.

File: components/deleteQuestion/index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+const StyledQuestion = styled.p`
+  text-align: center;
+  color: red;
+  font-size: 1.4rem;
+  margin-top: -10px;
+  padding: 10px;
+`;
+
+export default function DeleteQuestion({ handleDelete }) {
+  console.log("bin hier");
```
Comment (rueveydaakuel) [inline components/deleteQuestion/index.js:54]: don´t forget to remove the console.log()

**Review by rueveydaakuel — APPROVED** [review]
It's a very clean and readable code. I like it. I just have very minor comments, which is not a big deal :) You can fix it in the next PR

File: components/profileCard/index.js (hunk 1 above)
Comment (ChristophLeuker) [reply components/profileCard/index.js:57]: i need it because i got a form behind the listitem

File: components/deleteQuestion/index.js (hunk 2 above)
Comment (ChristophLeuker) [reply components/deleteQuestion/index.js:54]: removed ;)

**Comment by ChristophLeuker** [comment]
thanks for the review :)

**Pull request merged**